cmd/kubeadm/app/cmd/reset.go

```diff
@@ -27,7 +27,7 @@ import (
 	"github.com/spf13/cobra"
 
 	kubeadmapi "k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm"
-	"k8s.io/kubernetes/cmd/kubeadm/app/phases/kubeconfig"
+	kubeadmconstants "k8s.io/kubernetes/cmd/kubeadm/app/constants"
```

Comment:
I agree that some of the package aliases are bad, but I think this one makes sense as well.
This says explicitely that we're talking about kubeadm constants, no one else's constants.